ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -249,6 +335,130 @@ export default {
         },
       },
     },
+    {
+      id: 'DelubrumSav Guard Fiery Portent',
+      netRegex: NetRegexes.startsUsing({ source: 'Queen\'s Soldier', id: '583F' }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Soldat Der Königin', id: '583F' }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Soldat De La Reine', id: '583F' }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'クイーンズ・ソルジャー', id: '583F' }),
+      delaySeconds: (data, matches) => parseFloat(matches.duration) - 2.5,
+      durationSeconds: 5.5,
+      response: Responses.stopEverything('alarm'),
+    },
+    {
+      id: 'DelubrumSav Guard Icy Portent',
+      // Assuming you need to move for 3 seconds (duration of Pyretic from Fiery Portent)
+      netRegex: NetRegexes.startsUsing({ source: 'Queen\'s Soldier', id: '5840' }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Soldat Der Königin', id: '5840' }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Soldat De La Reine', id: '5840' }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'クイーンズ・ソルジャー', id: '5840' }),
+      delaySeconds: (data, matches) => parseFloat(matches.duration) - 2.5,
+      durationSeconds: 5.5,
+      response: Responses.moveAround('alert'),
+    },
+    {
+      id: 'DelubrumSav Guard Coat of Arms',
+      netRegex: NetRegexes.ability({ source: 'Aetherial Ward', id: '5820' }),
+      delaySeconds: (data, matches) => parseFloat(matches.duration) - 2.5,
+      suppressSeconds: 1,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Stop attacking',
+          de: 'Angriffe stoppen',
+          fr: 'Arrêtez d\'attaquer',
+          ja: 'ブロックしない側に攻撃',
+          cn: '攻击未格挡的方向',
+          ko: '공격 중지',
+        },
+      },
+    },
+    {
+      id: 'DelubrumSav Phantom Malediction Of Agony',
+      netRegex: NetRegexes.startsUsing({ source: 'Bozjan Phantom', id: '57BD', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Bozja-Phantom', id: '57BD', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Fantôme Bozjien', id: '57BD', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'ボズヤ・ファントム', id: '57BD', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'DelubrumSav Phantom Stuffy Wrath',
+      // Spawns after 57BA Summon, either North (-403.5) or South (-344.5)
+      // Casts 57C2 Undying Hatred
+      netRegex: NetRegexes.addedCombatantFull({ npcNameId: '9756' }),
+      durationSeconds: 5,
+      suppressSeconds: 1,
+      response: (data, matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          goSouth: {
+            en: 'Go South, Unavoidable Knocbkack',
+          },
+          goNorth: {
+            en: 'Go North, Unavoidable Knocbkack',
+          },
+        };
+
+        if (matches.y === -403.5)
```

Comment:
This will not work, as `matches.y` is a string.  (Someday, typescript will warn about this!)

Also, I think I don't like to trust that values are exact.  What's the other value? Can you list both of them in a comment and then do something like `parseFloat(matches.y) < 410` instead of `matches.y === -403.5`?

Reply:
The comment is under the id.
It always spawns on the y at -403.5 (North) or -344.5 (South).

Reply:
I put in the midway point between to spawns and chose < for North. It looks to me like the y values are inverted.

Reply:
Yeah +x is east and +y is south on all maps.  Thanks!